Backend/Meta-TV2-api/Meta-TV2-DataLayer/DataAccess.cs

```diff
@@ -99,4 +100,31 @@ public class DataAccess : IDataAccess
         await db.SaveChangesAsync();
         db.Dispose();
     }
+
+    public async void AddBlacklist(Blacklist obj) {
+        db.Add(obj);
+        await db.SaveChangesAsync();
+        db.Dispose();
+    }
+
+    public async Task<Optional<List<Blacklist>>> GetBlacklistedUsers() {
+        var query = from x in db.Blacklist select x;
+        var blacklists = await query.ToListAsync();
+        if (blacklists.Count == 0) {
+            return Optional<List<Blacklist>>.Empty();
+        }
+        return Optional<List<Blacklist>>.Result(blacklists);
+    }
+
+    public async Task<Optional<Blacklist>> GetBlacklistByAlias(string alias) {
+        var entry = await db.Blacklist.FindAsync(alias);

```

Comment:
I feel like this is a good approach. It's short and concise. This feels more readable to me than the method in "GetGroupsById". However this could be changed if I'm in minority.